### DISCUSSION TITLE
[UE]无法在ts里面直接读取DataTable某一行对应的结构数据？

### DISCUSSION THREAD

**Discussion by enixfu** [Tencent/puerts#1042] [Q&A] [answered]
如题，可以在ts里面获取DataTable某一行的数据，但类型是FTableRowBase，无法将其强制转换成对应的Struct，只能在C++或蓝图里面包装好获取接口，把对应的Struct数据传过来，但这样每加一类DataTable都要相应地加一个接口。请问有没有更加简洁的办法实现类似于C++模板函数这样的数据转换方式传递到ts里面？

**Comment by chexiongsheng** [comment, marked as answer]
看看这个issue对你有没帮助：https://github.com/Tencent/puerts/issues/986

**Reply by enixfu** [reply]
非常有用！十分感谢！
